<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admupdate</class>
 <widget class="QDialog" name="admupdate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: qconicalgradient(cx:0.548913, cy:1, angle:0, stop:0 rgba(0, 0, 103, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QLabel" name="label_invent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>151</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/img/update.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Name_label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="cat_lbl">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Product Cat:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Qty_edit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="product_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Product Name :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Cat_edit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="price_edit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Total_qty">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total_qty:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Price">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string> Price:</string>
   </property>
  </widget>
  <widget class="QLabel" name="qty_lbl">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Qty_Sold:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Total_edit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Save">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>UpdateTable</string>
   </property>
  </widget>
  <widget class="QLabel" name="qty_lbl_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Amount_Sold:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Amtedit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>290</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="desc_lbl_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Manufacture:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="manufact_edit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="desc_lbl_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Product Dsc:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="desc_edit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
